apps/labs/posts/automated-accessibility-testing-with-bokeh.md

```diff
@@ -0,0 +1,333 @@
+
+There were a few limitations to achieving all these objectives. The limitations
+were:
+
+- Complexities of the Bokeh codebase that required weeks of knowledge share
+  sessions.
+- The `<canvas>` element being something that couldn't properly be tested.
+- Initial difficulty in getting Playwright and Axe-core to run in the tests.
+- Time constraints.
+
+We were, however, able to achieve the following:
+
+- An audit and
+  [summary](https://github.com/bokeh/bokeh/discussions/14057#discussioncomment-10674506)
+  of ~113 doc examples that were run through axe-core.
+- Familiarity with the Bokeh codebase.
+- An implementation of tests that support Playwright and Axe-core.
+- A provision for creating new integration and unit tests and marking as
+  "accessibility-focused".
+
+Overall, our efforts towards going from 0 to 1 ended up looking like this [PR
+that adds Playwright.](https://github.com/bokeh/bokeh/pull/14032)
+
+## Progressive work going forward
+
+Going from 0 to 1 was a good start, but to go from 1 to n, there are more things
+that can be done in Bokeh. In the future, we could:
+
+- Add Integration snapshot tests that will be marked as known failures.
+- Add CI checks for accessibility tests. This would prevent commits from being
+  merged if they will change Bokeh data visualizations in a way that triggers
+  accessibility issues.
+- Couple things a little tighter with the existing convention. In Bokeh,
+  Javascript tests are run with a specific type of command: `node make test:suite_name`. There are certain commands for pairing types of tests
+  together, and these commands could be worked upon or modified as needed.
+- Have a labeling system to be able to run only "accessibility-focused" tests at
+  will. In Bokeh, you can run both individual and group tests using a search
+  string command. The syntax is: `node make test:suite_name -k "search string"`.
+  For labeling, you could prepend (or append) a string like
+  "accessibility-focused" to the code for the tests and run the matching search
+  string in the syntax to run all matching tests. This feature becomes even more
+  useful as `unit` and `integration` directories start to contain more
+  accessibility tests.
+- Write more tests.
+
+## References
+
+The testing syntax and conventions mentioned all reference the Bokeh guides on
+[running tests](https://docs.bokeh.org/en/latest/docs/dev_guide/testing.html#)
+and [writing
+tests](https://docs.bokeh.org/en/latest/docs/dev_guide/writing_tests.html)
+
+## Acknowledgement
+
+Several collaborative efforts led to the success of my internship work. I'm
+grateful to the following people for making the journey smoother: My mentor
+Gabriel, Tania and Pavithra for periodic information and pointers, Melissa for
+the periodic checkups and feedback loop, Bokeh maintainers Mateusz and Bryan,
+the Bokeh community for being welcoming, and Quansight for giving me an enabling
+environment to do some great work.
```

Comment:
You need to break up the list because you labeled the list as "following people" but then you included Bokeh community and Quansight, which are not people.

```suggestion
the periodic checkups and feedback loop, Bokeh maintainers Mateusz and Bryan.
I also want to thank the Bokeh community for being welcoming, and Quansight for giving me an enabling
environment to do some great work.
```